docs/cn/user-guide.md

```diff
@@ -0,0 +1,29 @@
+# Dumpling 使用手册
+
+[Dumpling](https://github.com/pingcap/dumpling) 是针对 MySQL/TiDB 数据库的 dump 成 SQL 文件工具。
+
+设计初衷是为了替代 [Mydumper](https://github.com/pingcap/mydumper), 所以基本用法可以参考 Mydumper, 
+当然在实现中没有完全照搬 Mydumper, 因此存在与 Mydumper 不同的用法。
+
+下表罗列了一些与 Mydumper 不同之处。
+
+| 主要参数 |     |
+| --------| --- |
+| -B 或 --database| 导出数据库名称|
```

Comment:
I think we should emphasize `-H` index here because it's quit different from mydumper.